src/subscription_manager/cert_sorter.py

```diff
@@ -289,9 +200,6 @@ def get_status_map() -> Dict[str, str]:
         # when function is called (not during start of application) due to
         # rhsm.service which can run for very long time
         status_map = {
-            VALID: _("Current"),
-            PARTIAL: _("Insufficient"),
-            INVALID: _("Invalid"),
             DISABLED: _("Disabled"),
             UNKNOWN: _("Unknown"),
```

Comment:
Another note: And this to "No registered" to be compatible with CLI. But no need to do it now.